test/base_renderer_test.rb

```diff
@@ -29,7 +33,35 @@ def render_hash
 
   test "properly handle assigns with symbol keys" do
     @context.assigns[:foo] = 'bar'
-    @template.source = {}
     assert_nothing_raised { render_hash }
   end
+
+  test "cache should be applied if no cache key is given" do
```

Comment:
Shouldn't this be "should not be applied" instead of "should be applied"? The next line is `@cache.should_not_receive(:fetch)`.